README.md

```diff
@@ -29,6 +29,32 @@ Jekyll Sass Converter comes bundled with Jekyll 2.0.0 and greater. For more
 information about usage, visit the [Jekyll Assets Documentation
 page](https://jekyllrb.com/docs/assets/).
 
+### Sass Implementations
+
+#### SassC
+
+By default, Jekyll Sass Converter uses [sassc](https://rubygems.org/gems/sassc)
+for Sass implmentation. `sassc` is based on LibSass, and
+[LibSass is deprecated](https://sass-lang.com/blog/libsass-is-deprecated).
+
+#### Sass Embedded
+
+[sass-embedded](https://rubygems.org/gems/sass-embedded) is a host for the
+[Sass embedded protocol](https://github.com/sass/embedded-protocol). The host
+runs [Dart Sass compiler](https://github.com/sass/dart-sass-embedded) as subprocess
+and communicates with it via message-passing.
```

Comment:
```suggestion
and communicates with the dart-sass compiler via message-passing.
```

but what is _message-passing_?

Reply:
@ntkme Could you elaborate what you mean by _message-passing_..?

Reply:
I copied that term from: https://sass-lang.com/blog/libsass-is-deprecated#what-about-portability-and-performance

Specifically, it means sending/receiving protobuf messages via stdio.